subscription/src/main/java/org/killbill/billing/subscription/engine/dao/SubscriptionDao.java

```diff
@@ -83,6 +83,8 @@
 
     public void createSubscriptionWithAddOns(List<DefaultSubscriptionBase> subscriptions, Map<UUID, List<SubscriptionBaseEvent>> initialEventsMap, InternalCallContext context);
```

Comment:
At the DAO level, maybe we can remove this redundant method (`createSubscriptionsWithAddOns` is a superset)?